CEXT-6421: update checkout starter kit docs for the 4-app split (#578)
AdobeDocs/commerce-extensibility: src/pages/starter-kit/checkout/totals-collector-install.md

<Fragment src="/_includes/checkout-configuration.md" />

> The `totals-collector` app has no custom installation step — unlike `shipping-method`, `payment-method`, and `tax-integration`, it doesn't create any Commerce entities, since it only implements webhook-based discount and fee actions. There's nothing to install or uninstall beyond the app itself.

Reviewer: ```suggestion
The `totals-collector` app does not have a custom installation step. Unlike the `shipping-method`, `payment-method`, and `tax-integration` apps, it does not create any Commerce entities. It only implements webhook-based discount and fee actions. There is nothing to install or uninstall beyond the app itself.
```

After installation, register one or more webhooks so that Adobe Commerce can call your app during quote totals collection:

> - **Discounts** — fires during cart totals collection. The app ships with 9 example discount actions and declares this webhook out of the box, defaulting to one of them; pick which example to use (or write your own) and set it as the webhook's runtime action. See [Discount use cases](totals-collector-use-cases.md#totals-collector-webhook) for registration details and response format.

Reviewer: ```suggestion
- **Discounts**. Fires during cart totals collection. The app ships with nine example discount actions and declares this webhook out-of-the-box, defaulting to one of them. Pick which example to use (or write your own) and set it as the webhook's runtime action. See [Discount use cases](totals-collector-use-cases.md#totals-collector-webhook) for registration details and response format.
```

> - **Custom fees** — fires during cart totals collection, after shipping costs and discounts are computed, giving your app access to the computed shipping cost and any shipping discount. The app doesn't declare this webhook out of the box; add your own runtime action and register it. See [Custom fees use cases](totals-collector-fees.md#custom-fees-webhook) for registration details and response format.

Reviewer: ```suggestion
- **Custom fees**. Fires during cart totals collection, after shipping costs and discounts are computed. Your app will have access to the computed shipping cost and any shipping discount. The app does not declare this webhook out-of-the-box. You must add your own runtime action and register it. See [Custom fees use cases](totals-collector-fees.md#custom-fees-webhook) for registration details and response format.
```

> - **Item prices** — fires during cart totals collection, immediately after the subtotal is computed. The app doesn't declare this webhook out of the box; add your own runtime action and register it. See [Item price use cases](totals-collector-item-prices.md#item-prices-webhook) for registration details and response format.

Reviewer: ```suggestion
- **Item prices**. Fires during cart totals collection, immediately after the subtotal is computed. The app does not declare this webhook out-of-the-box. Add your own runtime action and register it. See [Item price use cases](totals-collector-item-prices.md#item-prices-webhook) for registration details and response format.
```
